### ISSUE TITLE
Supporting Inuktitut Culture (iu-cans-ca) in Site Settings

### ISSUE THREAD

**Issue by easegui** [PiranhaCMS/piranha.core#1640]
Version: Piranha 8.4.2

Currently we are adding a new locale in our website for Inuktitut culture (iu-cans-ca), but we can not save it in settings because we are limited to 6 characters. Would you be able to support this one?

See image in reference:
![image](https://user-images.githubusercontent.com/1507050/119695886-a1eda580-be1c-11eb-8902-38c80104cce2.png)

We are using the culture value as well to display the information correctly from our application.

**Comment by tidyui** [comment]
Yes, unfortunately it requires a database migration since the `Culture` field has a fixed length of 6 characters in the database schema. What you could do in the meantime is add your culture into the `Description` field and then add a custom middleware component that executes **after** the piranha middleware which sets the thread culture according to the value set in the `Description` field. 

Best regards

**Comment by rizwanja** [comment]
@tidyui  Can we simply run a migration ourselves and update that entity's length to something greater than 6? Will that work or are there application level checks on that field apart from the DB field length also? Thank you in advance!
